homeassistant/components/zwave_js/discovery.py

```diff
@@ -89,6 +89,8 @@ class ZWaveDiscoverySchema:
     product_id: set[int] | None = None
     # [optional] the node's product_type must match ANY of these values
     product_type: set[int] | None = None
+    # [optional] the node's label must match ANY of these values
+    label: set[str] | None = None
```

Comment:
Is this label not at risk for breaking changes? I don't find it in the docs.

Reply:
I am assuming this is coming from the device config and is just undocumented (I spot checked a couple of our fixtures and they match). Do you think it would be safer to use the device config label instead?

Reply:
My assumption is correct: https://github.com/zwave-js/node-zwave-js/blob/master/packages/zwave-js/src/lib/node/Node.ts#L614-L616

Reply:
Maybe a bit safer to use the device config label attribute, yes. We do have the node label attribute in our model, though.

Reply:
Is the value of the label at risk for change?

Reply:
Well because it is coming from the device config, it theoretically is, but practically speaking I don't know that it will. Do you have an alternate parameter suggestion? I am fine with switching it to something more deterministic, I just don't know what that is short of putting in the effort to support firmware version ranges.

Reply:
now that I think about it some more, it doesn't hurt to try firmware version ranges. There's no discovery items using it right now, and that is what distinguishes this device from the other one that has a completely different configuration. We can use it for now and then if it breaks in the future we can reevaluate